src/pages/networks/actions/EditLocalPeerBtn.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FC } from "react";
+import { Button, Icon } from "@canonical/react-components";
+import usePanelParams from "util/usePanelParams";
+import type { LxdNetwork } from "types/network";
+import { useNetworkEntitlements } from "util/entitlements/networks";
+
+interface Props {
+  network: LxdNetwork;
+  localPeering: string;
+}
+
+const EditLocalPeerBtn: FC<Props> = ({ network, localPeering }) => {
+  const panelParams = usePanelParams();
+  const { canEditNetwork } = useNetworkEntitlements();
+
+  return (
+    <Button
+      key={`${localPeering}-edit`}
+      appearance="base"
+      hasIcon
+      onClick={() => {
+        panelParams.openEditLocalPeering(localPeering);
+      }}
+      title={
+        canEditNetwork(network)
+          ? "Edit local peer"
+          : "You do not have permission to edit this local peer"
```

Comment:
```suggestion
          ? "Edit local peering"
          : "You do not have permission to edit this local peering"
```
A bit more consistent naming?